library/java/org/chromium/net/AndroidCertVerifyResult.java

```diff
@@ -14,7 +14,7 @@ public final class AndroidCertVerifyResult {
   /**
    * The verification status. One of the values in CertVerifyStatusAndroid.
    */
-  private final int mStatus;
+  @CertVerifyStatusAndroid private final int mStatus;
```

Comment:
To educate myself, what is `@CertVerifyStatusAndroid` annotation for?

Reply:
The `@CertVerifyStatusAndroid` annotation is declared at [java/org/chromium/net/CertVerifyStatusAndroid.java](https://sourcegraph.com/github.com/envoyproxy/envoy-mobile/-/blob/library/java/org/chromium/net/CertVerifyStatusAndroid.java).

It basically makes use of Android's [IntDef](https://developer.android.com/reference/androidx/annotation/IntDef) annotation to say: this is a "special int" and it's values should be constrained to those declared by `@CertVerifyStatusAndroid`.

I've taken this approach from Chromium's code, I think it makes sense in this case (compared for example to using an enum) because we will need an actual int to pass it through the JNI layer.